frontend/src/scenes/instance/AsyncMigrations/AsyncMigrations.tsx

```diff
@@ -243,11 +275,26 @@ export function AsyncMigrations(): JSX.Element {
                     />
 
                     <Tabs activeKey={activeTab} onChange={(t) => setActiveTab(t as AsyncMigrationsTab)}>
-                        <TabPane tab="Management" key={AsyncMigrationsTab.Management} />
+                        <TabPane
+                            tab={`Management (${actionableMigrations.length})`}
+                            key={AsyncMigrationsTab.Management}
+                        />
+                        <TabPane
+                            tab={`Future Migrations (${futureMigrations.length})`}
+                            key={AsyncMigrationsTab.FutureMigrations}
+                        />
+                        <TabPane
+                            tab={`Completed Migrations (${completedMigrations.length})`}
+                            key={AsyncMigrationsTab.CompletedMigrations}
+                        />
                         <TabPane tab="Settings" key={AsyncMigrationsTab.Settings} />
                     </Tabs>
 
-                    {activeTab === AsyncMigrationsTab.Management ? (
+                    {[
+                        AsyncMigrationsTab.Management,
+                        AsyncMigrationsTab.FutureMigrations,
+                        AsyncMigrationsTab.CompletedMigrations,
```

Comment:
I'm not sold on the completed migrations tab. Example problem:

- I hit 'run' on the migration
- It completes successfully
- Autorefresh, my table is now empty and says "no migrations"

The UX is kind of awful.

What is the problem you're solving here with a separate completed migrations tab? Should this be part of this PR?

Reply:
It just seemed logical to separate completed ones out from both actionable and future migrations. This also solves the problem we discussed about having a lot of migrations show up in the management tab (especially if we move clickhouse migrations all there). We could show recently completed migrations for a bit in the management tab.

Reply:
I personally quite liked the full table - gave me a sense of everything in one place. I think seeing all my migrations is helpful compared to e.g. seeing installed and uninstalled plugins.

Maybe a checkbox to hide completed migrations is cleaner? Bonus points for something indicating that part of the table was collapsed (e.g. "5 completed migrations")

Reply:
The split came from Karl's request to break out future migrations that might confuse users and I figured that completed migrations, especially ones that were auto completed during the initial install could confuse users too, but since you both requested them to be together with active migrations and we currently don't have too many migrations I'll just remove for now to make this go faster.